mindjunkies/forums/models.py

```
@@ -0,0 +1,85 @@
+from django.db import models
+from django.conf import settings
+from django.utils.text import slugify
+from django.utils import timezone
```

Ruff (0.8.2)

`django.utils.timezone` imported but unused

Remove unused import: `django.utils.timezone`

(F401)